pkgs/development/libraries/linbox/default.nix

```diff
@@ -36,6 +35,16 @@ stdenv.mkDerivation rec {
     fflas-ffpack
   ];
 
+  patches = [
+    ./patches/linbox-pr256-part2.patch # TODO: Remove on 1.7.0 update
```

Comment:
Please give a short one-line explanation of the reason for this patch in a comment. Also include a link to the upstream PR.

Reply:
Sure.